### ISSUE TITLE
Previous and Next buttons are missing arrows on the site picker

### ISSUE THREAD

**Issue by markbiek** [Automattic/wp-calypso#98994]
Steps to reproduce:

1. Go to https://wordpress.com/setup/hosted-site-migration/sitePicker
2. Scroll to the bottom of the page.
3. The Previous and Next buttons are missing arrows.

![Image](https://github.com/user-attachments/assets/a0c9fd33-460b-4ff1-b40d-3791671d9a15)
